docker-compose.yml

```
@@ -1103,7 +1103,7 @@ services:
         aliases:
           - edx.devstack.frontend-app-learning
     ports:
-      - "2000:2000"
+      - "2010:2000"
```

Let's document here why we are doing this.
```suggestion
      - "2010:2000"  # avoid Codespaces port 2000 conflict
```

---

Changed and documented in every changed file about the resolution